src/pipelines_repository/adf-build/target.py

```diff
@@ -102,6 +102,9 @@ def _target_is_ou_path(self):
         responses = self.organizations.dir_to_ou(self.path)
         self._create_response_object(responses)
 
+    def _target_is_region_only(self):
```

Comment:
Im guessing you are wanting to check something here?

Reply:
This seemed to be the easiest way to get a targets section without a path defined.